### PR TITLE
Improves Ownable events

### LINKED ISSUE #1392: Improve Ownable events
`Ownable` currently has `OwnershipTransferred` and `OwnershipRenounced`: this is inconsistent with the rest of the project. We should get rid of `OwnershipRenounced`, and emit `OwnershipTransferred` both in the constructor (from address 0), and when renouncing (to address 0).

### PR DESCRIPTION
**Pull request by Aniket-Engg**
Fixes #1392

### CODE REVIEW

**Comment by nventuro** [comment]
Thanks a lot @Aniket-Engg!

Looks like the `Ownable` tests don't currently check for events :scream: Could you add those checks to this PR? The constructor one we can skip for now though, since that one is somewhat tricky (#1354 has the same problem).

**Comment by Aniket-Engg** [comment]
okay @nventuro

File: contracts/ownership/Ownable.sol [hunk 1]
```diff
@@ -19,7 +19,6 @@ contract Ownable {
    */
   constructor() public {
     _owner = msg.sender;
-    emit OwnershipTransferred(address(0), _owner);
```
Comment (nventuro) [inline contracts/ownership/Ownable.sol]: My bad for not being clear, I meant we should skip the test, not emitting the event :stuck_out_tongue:

Comment (Aniket-Engg) [reply contracts/ownership/Ownable.sol]: okay :smile: , no problem. Will add that.

**Pull request merged**